client/devdocs/design/index.jsx

```diff
@@ -68,6 +68,8 @@ export default React.createClass( {
 	render() {
 		return (
 			<div className="design-assets" role="main">
+				<h1>Design Reference</h1>
```

Comment:
What do you think about dropping the titles? I don't think it adds much but takes a lot of space.